radixdlt-core/radixdlt/src/main/java/com/radixdlt/statecomputer/substatehash/BerkeleySubstateAccumulatorHashStore.java

```diff
@@ -153,29 +187,23 @@ public void process(
     for (REStateUpdate reStateUpdate : txn.stateUpdates().toList()) {
       subStateBytes = Arrays.concatenate(subStateBytes, getBytes(reStateUpdate));
```

Comment:
@dhedey should I use a 32 zero byte array as the `subStateBytes` initial value too?